packages/react/src/themes/teams/components/RadioGroup/radioGroupItemStyles.ts

```diff
@@ -12,33 +12,55 @@ const radioStyles: ComponentSlotStylesInput<
 > = {
   root: ({ props }): ICSSInJSStyle => ({
     outline: 0,
+    display: 'inline-flex',
+    alignItems: 'center',
+    cursor: props.disabled ? 'default' : 'pointer',
     ...(!props.vertical && {
       display: 'inline-block',
     }),
   }),
 
   label: ({ props: p, variables: v }): ICSSInJSStyle => ({
-    cursor: 'pointer',
-    display: 'inline-flex',
-    alignItems: 'baseline',
-    fontWeight: 400,
-    minHeight: '2.5rem',
     backgroundColor: 'transparent',
+    color: p.checked ? v.labelColorChecked : v.labelColorDefault,
+    ':hover': {
+      color: v.labelColorChecked,
```

Comment:
if we expect hover and focus behavior to apply the same style it'd be better to extract the styles in one object and reuse it